### PR TITLE
Port update package version improvements 

### PR DESCRIPTION
**Pull request by angrybayblade**
## Proposed changes

Port improvements from main repo

## Fixes

+ 

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: scripts/update_package_versions.py [hunk 1]
```diff
@@ -182,8 +182,8 @@ def split_hashes_by_type(all_hashes: Dict[str, str]) -> Dict[str, Dict[str, str]
         "skills": {},
     }  # type: Dict[str, Dict[str, str]]
     for key, value in all_hashes.items():
-        if "fetchai" not in key:
-            print("Non-fetchai packages not allowed!")
+        if "fetchai" not in key and "open_aea" not in key:
+            print("Non (fetchai|open_aea) packages not allowed!")
```
Comment (angrybayblade) [inline scripts/update_package_versions.py:186]: Allowing both fetchai and open_aea authored packages

File: scripts/update_package_versions.py [hunk 2]
```diff
@@ -192,12 +192,19 @@ def split_hashes_by_type(all_hashes: Dict[str, str]) -> Dict[str, Dict[str, str]
...
 def get_configuration_file_path(type_: str, name: str) -> Path:
     """Get the configuration file path."""
+
     fp = os.path.join("packages", "fetchai", type_, name, TYPE_TO_CONFIG_FILE[type_])
     if os.path.isfile(fp):
         return Path(fp)
+
+    fp = os.path.join("packages", "open_aea", type_, name, TYPE_TO_CONFIG_FILE[type_])
+    if os.path.isfile(fp):
+        return Path(fp)
```
Comment (angrybayblade) [inline scripts/update_package_versions.py:202]: Same as above

File: scripts/RELEASE_PROCESS.md [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
+2. Determine the next AEA version 
+   Create new release branch named "feature/release-{new-version}, switch to this branch"
+   Run `python scripts/bump_aea_version.py --new-version NEW_VERSION_HERE`. Commit if satisfied.
 
 3. Bump plugin versions if necessary by running `python scripts/update_plugin_versions.py --update "PLUGIN_NAME,NEW_VERSION"`. Commit if satisfied.
 
 4. Check the protocols are up-to-date by running `python scripts/generate_all_protocols.py`. Commit if changes occurred.
 
-5. [CURRENTLY SKIPPED] Bump all the packages to their latest versions by running `python scripts/update_package_versions.py`.
+5. Bump all the packages to their latest versions by running `python scripts/update_package_versions.py`.
```
Comment (DavidMinarsch) [inline scripts/RELEASE_PROCESS.md:14]: please revert, we don't have this equivalent atm

File: scripts/RELEASE_PROCESS.md [hunk 4]
```diff
@@ -28,12 +30,12 @@
 13. Publish to PyPI with twine: `twine upload dist/*`. Optionally, publish to Test-PyPI with twine:
 `twine upload --repository-url https://test.pypi.org/legacy/ dist/*`.
 
-14. Repeat 11. & 12. for each plugin (use `python setup.py sdist bdist_wheel`).
+14. Repeat 11. & 12. for each plugin.
```
Comment (DavidMinarsch) [inline scripts/RELEASE_PROCESS.md:33]: please revert

File: scripts/RELEASE_PROCESS.md [hunk 5]
```diff
@@ -28,12 +30,12 @@
...
 
-15. Make clean environment and install release from PyPI: `pip install open-aea[all] --no-cache`.
+15. Make clean environment and install release from PyPI: `pip install aea[all] --no-cache`.
 
-16. Release packages into registry: `python scripts/deploy_to_registry.py`.
+16. Release packages into registry: `python scripts/deploy_to_registry.py`. Run it several times till all packages updated, cause some dependencies order.
```
Comment (DavidMinarsch) [inline scripts/RELEASE_PROCESS.md:37]:
```suggestion
16. [CURRENTLY SKIPPED] Release packages into registry: `python scripts/deploy_to_registry.py`. If necessary, run it several times until all packages are updated.
```

File: scripts/generate_ipfs_hashes.py [hunk 6]
```diff
@@ -223,12 +235,16 @@ def __init__(self, timeout: float = 15.0):
     def __enter__(self) -> None:
         """Run the ipfs daemon."""
         self.process = subprocess.Popen(  # nosec
-            ["ipfs", "daemon", "--offline"],
-            stdout=subprocess.PIPE,
-            env=os.environ.copy(),
+            ["ipfs", "daemon"], stdout=subprocess.PIPE, env=os.environ.copy(),
```
Comment (DavidMinarsch) [inline scripts/generate_ipfs_hashes.py:238]: Please read the flag you accidentally deleted

File: scripts/RELEASE_PROCESS.md [hunk 7]
```diff
@@ -28,12 +30,12 @@
...
 
-17. Create and push Docker images `user-image` and `develop-image`.
+17. AEA develop docker image don automatically with CI with develop to main PR.
```
Comment (DavidMinarsch) [inline scripts/RELEASE_PROCESS.md:39]:
```suggestion
17. The `user-image` and `develop-image` are update via CI.
```

**Review by DavidMinarsch — APPROVED** [review]
LGTM

**Pull request merged**